### PR TITLE
Update to MSGETACLSTATUS

### PR DESCRIPTION
**Pull request by begoldsm**
This maintains the previous functionality for PowerShell callers and also adds in a new optional property called "permission"

### CODE REVIEW

File: arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json [hunk 1]
```diff
@@ -1161,8 +1161,14 @@
           "type": "string",
           "description": "the user owner, an AAD Object ID."
         },
+		"permission": {
+          "type": "integer",
+		  "format": "int32",
+          "description": "The octal representation of the unnamed user, mask and other permissions."
+        },
```
Comment (John-Hart) [inline arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json]: can you fix the formatting here?

**Comment by begoldsm** [comment]
@John-Hart good catch! Tabbing fixed back to 2 spaces (curse you notepad++!)

File: arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json [hunk 2]
```diff
@@ -116,6 +116,72 @@
...
+              "SETEXPIRY"
+            ]
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
```
Comment (amarzavery) [inline arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json]: no empty descriptions allowed

File: arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json [hunk 3]
```diff
@@ -872,7 +938,91 @@
...
+              "REMOVEDEFAULTACL"
+            ]
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
```
Comment (amarzavery) [inline arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json]: no empty description

File: arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json [hunk 4]
```diff
@@ -872,7 +938,91 @@
...
+              "REMOVEACL"
+            ]
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
```
Comment (amarzavery) [inline arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json]: same as above

**Comment by begoldsm** [comment]
@amarzavery I will put in descriptions for the new stuff in this PR, and then we will need to have another documentation specific PR later to get the rest of the specs to be fully documented.

**Comment by amarzavery** [comment]
sure

File: arm-datalake-store/account/2015-10-01-preview/swagger/account.json [hunk 5]
```diff
@@ -679,6 +715,69 @@
       },
       "description": "Data Lake Store firewall rule list information."
     },
+    "EncryptionIdentity": {
```
Comment (amarzavery) [inline arm-datalake-store/account/2015-10-01-preview/swagger/account.json]: I dont see this model being referenced anywhere.

Comment (begoldsm) [inline arm-datalake-store/account/2015-10-01-preview/swagger/account.json]: It is being referenced in DataLakeStoreAccount

File: arm-datalake-store/account/2015-10-01-preview/swagger/account.json [hunk 6]
```diff
@@ -757,6 +886,10 @@
           "readOnly": true,
           "description": "the account subscription ID."
         },
+        "identity": {
```
Comment (begoldsm) [inline arm-datalake-store/account/2015-10-01-preview/swagger/account.json]: Right here is where it is being reference @amarzavery. I just double checked that I didn't spelly it wrong.

File: arm-datalake-store/account/2015-10-01-preview/swagger/account.json (hunk 5 above)
Comment (amarzavery) [inline arm-datalake-store/account/2015-10-01-preview/swagger/account.json]: sorry. I got it.

**Pull request merged**